gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/JavaServiceDistributionExtension.java

```diff
@@ -148,6 +151,14 @@ public final void enableManifestClasspath(boolean newEnableManifestClasspath) {
         this.enableManifestClasspath.set(newEnableManifestClasspath);
     }
 
+    public final Provider<Boolean> getEnableAlwaysPreTouch() {
+        return enableAlwaysPreTouch;
+    }
+
+    public final void enableAlwaysPreTouch(boolean alwaysPreTouch) {
```

Comment:
I recall that gradle does something special with `getEnableAlwaysPreTouch` and `setEnableAlwaysPreTouch` which impacts the way methods can be called.

I think `void enableAlwaysPreTouch(boolean alwaysPreTouch)` allows us to use `enableAlwaysPreTouch true` where `setEnableAlwaysPreTouch` would allow us to use `enableAlwaysPreTouch = true`

Reply:
The build folks would have a better idea what best practice looks like for these things than I do

Reply:
I added both types of setter - seems like some of the other options do this. I'll check the some build folks too